config/routes.php

```diff
@@ -1,4 +1,4 @@
 <?php
-use Cake\Routing\Router;
-
-Router::extensions(['pdf']);
```

Comment:
@Marlinc  Not necessarily. With plugin routes being loaded after the application rules in the default application template, this line never had any effect, it only would if one had moved the `Router::plugins()` call before the application rules, or at least before those that are ment to support the `.pdf` extension, and I while I may go out on a limb here, I'd say that 9+ out of 10 people haven't done that, but just wondered why things aren't working, and eventually registered the extension manually.

However, for those that actually did that, it would be a BC break indeed, so if that is a concern, this could be done in two steps, first just update the docs and add a note in the bootstrap file that mentions that this has no effect when loaded too late, and then later on remove the bootstrap when BC breaks are acceptable.